templates/base.html

```diff
@@ -1,101 +1,107 @@
-{% load render_bundle from webpack_loader %}
+{% load render_bundle from webpack_loader %} 
 {% load static %}
-
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+
+<head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>
-      {% block title %}
-      paneldata.org
-      {% endblock title %}
+        {% block title %}paneldata.org
+        {% endblock title %}
     </title>
-
-     <!-- Webpack bundle: "index" CSS -->
-    {% render_bundle 'index' 'css' %}
-
-    {% block styles %}
+    <!-- Webpack bundle: "index" CSS -->
+    {% render_bundle 'index' 'css' %} 
+    {% block styles %} 
     {% endblock styles %}
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media 
+        queries -->
+<!-- WARNING: Respond.js doesn't work if you view the page via file://
+-->
+<!--[if lt IE 9]>
+<![endif]-->
+</head>
 
-    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
-    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
-    <!--[if lt IE 9]>
-      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
```

Comment:
This got lost in translation. I guess no one should use IE 9 anyway nowadays.
So just remove the whole block that is left?